### ISSUE TITLE
Image format documentation

### ISSUE THREAD

**Issue by jburdy** [dukelec/mb#14]
Hi,
Were are the documentation about requirements to upload image in the Princube ? We don't know how to create correct format (transparency, etc...).
Thanks,

**Comment by dukelec** [comment]
As the API Document says, you need to call mbc.conv in /js/workers/mbc.js for image format conversion, convert png (which other formats may also support) to the private mbd format, and you can upload it to the printer.
The mbd file names are numbers, starting from 0, uploads more than one file is a multi-line print.

Parameter density is recommended not to set it too high, around 50 is recommended. Other parameters can use default values.

**Comment by dukelec** [comment]
If you're just using this APP and not secondary development, you can use the APP to open almost any image format (GIF animations only show the first frame).

**Comment by chonigman** [comment]
Is there a way to call mbc.conv from outside the app? It would be very useful to be able to convert and upload images programmatically.

**Comment by dukelec** [comment]
@chonigman Yes, I just added instructions and demo code for you, please refer to the headless conversion in the dev document.

**Comment by chonigman** [comment]
@dukelec Amazing. Thank you so much.
